### PR TITLE
First stab: security part of the architecture intent template

### PR DESCRIPTION
**Pull request by JonathanKamensDVA**

### CODE REVIEW

File: platform/engineering/collab-cycle/architecture-intent-meeting.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 __*** DRAFT *** DRAFT *** DRAFT ***__
 
-The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering and security standards and lowers the potential for launch-blocking issues later in the development cycle.
```
Comment (little-oddball) [inline platform/engineering/collab-cycle/architecture-intent-meeting.md:5]: I believe the name was been changed from VSP to VA.gov Platform a while back. @andreahewitt-odd would know better for sure.

File: platform/engineering/collab-cycle/architecture-intent-meeting.md [hunk 2]
```diff
@@ -17,20 +17,19 @@ You should schedule an Architecture Intent meeting if any of these apply:
...
 - Your change requires complex coordination across teams or you need support to coordinate across teams.
 - You're gathering new Personally Identifiable Information ([PII]) and/or Protected Health Information ([PHI]) or saving PII/PHI in a new place.
 - There are cost, performance or security implications to your change.
 
-[PII]: https://en.wikipedia.org/wiki/Personal_data
-[PHI]: https://en.wikipedia.org/wiki/Protected_health_information
-
 ## Preparing for the Architecture Intent meeting
 
 You should write up your Architecture Intent using the template below.  The Governance Team will need at least **2 business days** to review your materials.
```
[the remark this answers is not in the document]
Comment (JonathanKamensDVA) [reply platform/engineering/collab-cycle/architecture-intent-meeting.md:29]: I think we mean Governance Team in the broader sense here of everybody involved in the the collaboration cycle from the governance side, not in the specific sense of the folks in Governance at Oddball. I think one or more of Don, Joe, Tom, and I will be reviewing these, at least to start.

File: platform/engineering/collab-cycle/architecture-intent-meeting.md [hunk 3]
```diff
@@ -75,11 +75,62 @@ Some of the items below may not apply to your work--that's okay.  You may not be
     + List scope of any feature flags
     + Identify other teams to coordinate with
     + Describe rollback plan
```
[the remark this answers is not in the document]
Comment (JonathanKamensDVA) [reply platform/engineering/collab-cycle/architecture-intent-meeting.md:77]: I will add an item about potential infrastructure needs.

DB changes is addressed a bit above this in the document (see the "Data storage" bullet point).

**Review by joeniquette — APPROVED** [review]
authentication and identity related sections look good to me.

**Pull request merged**